### PR TITLE
Tweak to fees and funding to clarify 'right immigration status'

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Tweak to fees and funding to clarify 'right immigration status'

### CODE REVIEW

**Comment by laurahermionetennant92** [comment]
@philspray if you have time it'd be great if you review this in Ellie's absence :)

**Review by philspray — DISMISSED** [review]
Looks ok to me

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md [hunk 1]
```diff
@@ -53,8 +53,9 @@ keywords:
   - Settlement Scheme
 ---
 
-If you’re applying for teacher training as a non-UK citizen, you may be able to get some financial help from the UK government. The information on this page is for:
+If you’re applying for teacher training as a non-UK citizen, you may be able to get some financial support from the UK government. Your eligibility for support will depend on your [immigration status](#check-your-immigration-status-and-eligibility-for-financial-support) and choice of subject. Most non-UK citizens without some type of settled status in the UK will not be eligible, unless they train to teach languages or physics.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md:56]:
```suggestion
If you’re applying for teacher training as a non-UK citizen, you may be able to get some financial support from the UK government.

Your eligibility for support will depend on your [immigration status](#check-your-immigration-status-and-eligibility-for-financial-support) and choice of subject. Most non-UK citizens without some type of settled status in the UK will not be eligible, unless they train to teach languages or physics.
```
maybe just split this para up to make it a bit more readable?

**Review by EllieNodder — APPROVED** [review]
Looks good to me!

**Review by philspray — APPROVED** [review]
Looks ok to me

**Pull request merged**